src/java.desktop/share/classes/javax/swing/JSplitPane.java

```diff
@@ -418,12 +418,18 @@ public String getUIClassID() {
 
     /**
      * Sets the size of the divider.
+     * @implNote Divider sizes < 1 are ignored.
+     *           {@code SplitPane.dividerSize} L&amp;F specific value
+     *           will instead be used.
```

Comment:
This isn't the same as ignoring it. Suppose we do
setDividerSize(5);
setDividerSize(-1);

what is the value after both of these - per your ignoring code 5, but per your doc it will be whatever the default was before you started to change it.

I think it sufficient to say it will be ignored and drop the 2nd clause.

Reply:
Ok. Modified javadoc and CSR

Reply:
My apologies, but because I focused on the rest of it, I overlooked you had made this an implNote.
That seems odd. Do you really intend to imply that some other implementation of Java should be allowed
to do something with negative values ? If so that would be a compatibility issue ..

Reply:
Yes, actually I was thinking in that sense, as I was not sure if other impl. will allow -ve values and just wanted to mention that our impl dont.
But, I guess as you pointed, we should make it disallowed for all. Modified PR and CSR.